ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/util/CommandParserTest.java

```diff
@@ -0,0 +1,106 @@
+    List<SqlCommand> commands = CommandParser.parse("CREATE STREAM riderLocations (profileId VARCHAR, latitude DOUBLE, longitude DOUBLE) WITH (kafka_topic='locations', value_format='json', partitions=1);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('c2309eec', 37.7877, -122.4205);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('18f4ea86', 37.3903, -122.0643);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('4ab5cbad', 37.3952, -122.0813);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('8b6eae59', 37.3944, -122.0813);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('4a7c7b41', 37.4049, -122.0822);\n"
+        + "INSERT INTO riderLocations (profileId, latitude, longitude) VALUES ('4ddad000', 37.7857, -122.4011);");
+    assertThat(commands.size(), is(7));
+    assertThat(commands.get(0), instanceOf(SqlStatement.class));
+    assertThat(commands.get(0).getCommand(), is("CREATE STREAM riderLocations (profileId VARCHAR, latitude DOUBLE, longitude DOUBLE) WITH (kafka_topic='locations', value_format='json', partitions=1);"));
+    assertThat(commands.get(1), instanceOf(SqlInsertValues.class));
+    assertThat(((SqlInsertValues) commands.get(1)).getSourceName(), is("RIDERLOCATIONS"));
+    assertThat(((SqlInsertValues) commands.get(1)).getColumns().size(), is(3));
+    assertThat(((SqlInsertValues) commands.get(1)).getColumns().get(0), is("PROFILEID"));
+    assertThat(((SqlInsertValues) commands.get(1)).getColumns().get(1), is("LATITUDE"));
+    assertThat(((SqlInsertValues) commands.get(1)).getColumns().get(2), is("LONGITUDE"));
+    assertThat(((SqlInsertValues) commands.get(1)).getValues().size(), is(3));
+    assertThat(toFieldType(((SqlInsertValues) commands.get(1)).getValues().get(0), Type.STRING), is("c2309eec"));
+    assertThat(toFieldType(((SqlInsertValues) commands.get(1)).getValues().get(1), Type.DECIMAL), is(new BigDecimal(37.7877).setScale(4, RoundingMode.DOWN)));
+    assertThat(toFieldType(((SqlInsertValues) commands.get(1)).getValues().get(2), Type.DECIMAL), is(new BigDecimal(-122.4205).setScale(4, RoundingMode.DOWN)));
+  }
+
+  @Test
+  public void shouldParseCreateConnectorStatement() {
+    final String command = "CREATE SOURCE CONNECTOR `jdbc-connector` WITH(\n"
+        + "    \"connector.class\"='io.confluent.connect.jdbc.JdbcSourceConnector',\n"
+        + "    \"connection.url\"='jdbc:postgresql://localhost:5432/my.db',\n"
+        + "    \"mode\"='bulk',\n"
+        + "    \"topic.prefix\"='jdbc-',\n"
+        + "    \"table.whitelist\"='users',\n"
+        + "    \"key\"='username');";
+    List<SqlCommand> commands = CommandParser.parse(command);
+    assertThat(commands.size(), is(1));
+    assertThat(commands.get(0), instanceOf(SqlConnectorStatement.class));
+    assertThat(commands.get(0).getCommand(), is(command));
+  }
+}
```

Comment:
We can do this as a follow-up PR but it'd be good to add more unit test coverage here for different types of statements: create sink connector, different types of insert values statements (with and without explicit field names, with quoted field names, with quoted stream name), insert into ... as select (to ensure it's not confused with insert values), unsupported statement types (to ensure a sane error message is returned), etc.